source/crud/tests/v2/updateWithPipelines.yml

```diff
@@ -16,6 +16,14 @@ tests:
         arguments:
             filter: { y: 1 }
             update: [ { $replaceRoot: { newRoot: "$t" } }, { $addFields: { foo: 1 } } ]
+        result:
+          deletedCount: 0
+          insertedCount: 0
+          insertedIds: {}
```

Comment:
`deletedCount`, `insertedCount` and `insertedIds` are for a BulkWriteResult. Since this operation is `updateOne`, just the UpdateResult fields below will suffice.

Reply:
I have removed `deletedCount`, `insertedCount` and `insertedIds`.